kepler/app.py

```diff
@@ -30,9 +34,33 @@ def register_blueprints(app):
     app.register_blueprint(item_blueprint)
     app.register_blueprint(layer_blueprint)
     app.register_blueprint(marc_blueprint)
+    app.logger.info("Blueprints registered")
 
 
 def register_errorhandlers(app):
     def handle_unsupported_format(error):
         return '', 415
     app.errorhandler(UnsupportedFormat)(handle_unsupported_format)
+    app.logger.info("Error Handlers registered")
+
+
+def register_loggers(app):
+    formatter = logging.Formatter(
+        "[%(asctime)s] [%(name)s] [%(levelname)s] [%(message)s]",
```

Comment:
looking at this now on Heroku, we can probably skip logging time as they have a wrapper that already does that.